src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -435,11 +433,17 @@ export default class FilingHistoryList extends Mixins(
     return sessionStorage.getItem('BUSINESS_ID')
   }
 
+  /** Whether to show the history panel. */
+  get showHistoryPanel () {
+    return (this.historyItems.length > 0)
+  }
```

Comment:
It's a getter so it's a little faster than the old code (ie, cached), while the old code was called twice for every rended. but more importantly, use of this getter (for array length check) is similar to Todo List.